Casks/font/font-n/font-nanum-square-neo.rb

```diff
@@ -0,0 +1,21 @@
+cask "font-nanum-square-neo" do
+    version :latest
+    sha256 :no_check
+  
+    url "https://hangeul.naver.com/hangeul_static/webfont/zips/nanum-square-neo.zip"
+    name "NanumSquare Neo"
+    desc "Naver NanumSquare Neo font"
+    homepage "https://hangeul.naver.com/font/"
```

Comment:
```suggestion
  homepage "https://campaign.naver.com/nanumsquare_neo/"
```

Is this a better homepage?

Reply:
Yes. I changed the URLs.